src/DisplayBoard.cpp

```diff
@@ -101,3 +99,14 @@ namespace
 
 }
 
+std::ostream& operator<<(std::ostream& os,const Board& b)
```

Comment:
Do you really need this operator<<? Why not move below implementation into Board::display()?
Only make drawBoard function accessible here, so get it out of the unnamed namespace.
Some code could be removed then like DisplayInterface. The less code we have the better it is.

Reply:
I am closing it, because it is outdated.